robottelo/ui/base.py

```diff
@@ -82,8 +82,14 @@ def navigate_to_entity(self):
         """Perform navigation to main page for specific entity"""
         raise NotImplementedError('Subclasses must implement navigator method')
 
-    def search(self, element_name):
-        """Uses the search box to locate an element from a list of elements."""
+    def search(self, element):
+        """Uses the search box to locate an element from a list of elements.
+
+        :param element: either element name or a tuple, containing element name
+        as a first element and all the rest variables required for element
```

Comment:
Sigh. I wish travis failed builds when not only errors, but also warnings are present in the docstrings :(